ui/src/components/Liquidity/LiquidityPool/ItemLiquidityPool.jsx

```diff
@@ -19,7 +18,7 @@ const ItemLiquidityPool = ({
   const { state } = useApplicationContext();
   const { assets } = state;
 
-  const [error, setError] = useContext(ErrorContext);
+  const [_, setError] = useContext(ErrorContext);
```

Comment:
I'd prefer not using just `_` as it pollutes the local namespace.

```suggestion
  const [_error, setError] = useContext(ErrorContext);
```